website/content/api-docs/acl/templated-policies.mdx

```diff
@@ -0,0 +1,189 @@
+| `GET`  | `/acl/templated-policy/name/:name` | `application/json` |
+
+The following table shows this endpoint's support for
+[blocking queries](/consul/api-docs/features/blocking),
+[consistency modes](/consul/api-docs/features/consistency),
+[agent caching](/consul/api-docs/features/caching), and
+[required ACLs](/consul/api-docs/api-structure#authentication).
+
+| Blocking Queries | Consistency Modes | Agent Caching | ACL Required |
+| ---------------- | ----------------- | ------------- | ------------ |
+| `NO`            | `none`             | `none`        | `acl:read`   |
+
+The corresponding CLI command is [`consul acl templated-policy read -name=<string>`](/consul/commands/acl/templated-policy/read#name).
+
+### Path parameters
+
+- `name` `(string: <required>)` - Specifies the name of the ACL policy to read.
+
+### Sample request
+
+```shell-session
+$ curl --request GET http://127.0.0.1:8500/v1/acl/templated-policy/name/builtin/service
+```
+
+### Sample response
+
+```json
+{
+  "TemplateName": "builtin/service",
+  "Schema": "{\n\t\"type\": \"object\",\n\t\"properties\": {\n\t\t\"name\": { \"type\": \"string\", \"$ref\": \"#/definitions/min-length-one\" }\n\t},\n\t\"required\": [\"name\"],\n\t\"definitions\": {\n\t\t\"min-length-one\": {\n\t\t\t\t\"type\": \"string\",\n\t\t\t\t\"minLength\": 1\n\t\t}\n\t}\n}",
+  "Template": "\nservice \"{{.Name}}\" {\n\tpolicy = \"write\"\n}\nservice \"{{.Name}}-sidecar-proxy\" {\n\tpolicy = \"write\"\n}\nservice_prefix \"\" {\n\tpolicy = \"read\"\n}\nnode_prefix \"\" {\n\tpolicy = \"read\"\n}"
+}
+```
+
+## Preview a templated policy
+
+The `preview` endpoint shows the policy created from a templated policy. The endpoint helps you understand what templated variables are required or are missing from your request.
+
+| Method | Path          | Produces           |
+| ------ | ------------- | ------------------ |
+| `POST`  | `/acl/templated-policy/preview/:name` | `application/json` |
+
+The following table shows this endpoint's support for
+[blocking queries](/consul/api-docs/features/blocking),
+[consistency modes](/consul/api-docs/features/consistency),
+[agent caching](/consul/api-docs/features/caching), and
+[required ACLs](/consul/api-docs/api-structure#authentication).
+
+| Blocking Queries | Consistency Modes | Agent Caching | ACL Required |
+| ---------------- | ----------------- | ------------- | ------------ |
+| `NO`             | `none`            | `none`        | `acl:read`  |
+
+The corresponding CLI command is [`consul acl templated-policy preview`](/consul/commands/acl/templated-policy/preview).
+
+### Path parameters
+
+- `name` `(string: <required>)` - Specifies the name of the ACL templated policy to preview.
+
+### Query parameters
+
+- `ns` `(string: "")` <EnterpriseAlert inline /> - Specifies the namespace of the acl templated policy will be used.
+  You can also [specify the namespace through other methods](#methods-to-specify-namespace).
```

Comment:
```suggestion
- `ns` `(string: "")` <EnterpriseAlert inline /> - Specifies the namespace that the policy created from the template applies to.
  You can also [specify the namespace through other methods](#methods-to-specify-namespace).
```
Is that correct? Or are you specifying the namespace where Consul retrieves the specified template?